components/concourse-task-toolbox/Dockerfile

```diff
@@ -27,9 +27,12 @@ RUN apk add --update \
 	less \
 	python \
 	py-pip \
+	python3 \
```

Comment:
Just a thought, would we need [python3.7](https://packages.ubuntu.com/disco/python3.7)?

Reply:
The only particularly "new" feature we're using is I think PEP 3102. Since that was made in 2006, it landed in Python 3.0.

Reply:
https://stackoverflow.com/a/4255133 is a pretty good explanation of that PEP btw, it's used for genResourceID